.github/workflows/ci.yml

```diff
@@ -14,18 +14,41 @@ env:
   try-runtime-uri: wss://eden-rpc.dwellir.com:443
 
 jobs:
-  test:
+  clippy_lint:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: simonsso/setup-rust-toolchain@3b513b9b4098bc5ff24c930d279f9501375a1a09
+        with:
+          toolchain: ${{ env.nightly }}
+          target: ${{ env.target }}
+          components: clippy
```

Comment:
Do we still need the highlighted lines any more?

Reply:
It should not be the fork but the source of the fork.